WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1175,7 +1158,8 @@ private String migrateLegacyDraft(String content) {
             while (matcher.find()) {
                 String stringUri = matcher.group(1);
                 Uri uri = Uri.parse(stringUri);
-                MediaFile mediaFile = queueFileForUpload(uri, getContentResolver().getType(uri), null, "failed");
+                MediaFile mediaFile = WPStoreUtils.fromMediaModel(queueFileForUpload(uri,
```

Comment:
Added it here: https://github.com/wordpress-mobile/WordPress-Android/issues/5214#issuecomment-280636970